easybuild/easyconfigs/b/BamTools/BamTools-2.5.1-iccifort-2019.1.144-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,18 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+name = 'BamTools'
+version = '2.5.1'
+
+homepage = 'https://github.com/pezmaster31/%(namelower)s'
+description = "BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files."
+
+toolchain = {'name': 'iccifort', 'version': '2019.1.144-GCC-8.2.0-2.31.1'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/pezmaster31/%(namelower)s/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4abd76cbe1ca89d51abc26bf43a92359e5677f34a8258b901a01f38c897873fc']
+
+builddependencies = [('CMake', '3.13.3')]
```

Comment:
The foss-2017b version has pkg-config as a build dep, isn't that needed?

Reply:
@boegel `pkg-config` is still required. I will push a PR that will add the dependency to the existing easyconfig files that require it.